src/components/AccordionList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { useTestIds } from "..";
+import { Accordion, AccordionProps, AccordionSize } from "./Accordion";
+
+interface AccordionListProps {
+    accordions: AccordionProps[];
+    /** Allows multiple accordions to be expanded simultaneously (enabled by default) */
+    allowMultipleExpanded?: boolean;
+    size?: AccordionSize;
+}
+
+export function AccordionList({ accordions, size, allowMultipleExpanded = true }: AccordionListProps) {
+    const [currentSelectedIndex, setCurrentSelectedIndex] = useState<number>();
+    const tid = useTestIds({}, "accordionList");
+
+    return (<>
+        {accordions.map((accordionProps, index, arr) => (
+            <Accordion
+                {...accordionProps}
+                {...tid[index]}
+                key={index}
+                size={size}
+                bottomBorder={index === arr.length - 1}
+                allowMultipleExpanded={allowMultipleExpanded}
+                index={index}
+                currentSelectedIndex={currentSelectedIndex}
```

Comment:
**question:** I wondering if we can hide some of the `allowMultipleExpanded`, `index`, `currentSelectedIndex` logic from the single Accordion and have the list handle this?

Reply:
Specifically, I'm hoping we can make Accordion component unaware of other accordions around it, and instead have the AccordionList be responsible for making sure the `allowMultipleExpanded` property is respected.